packages/patternfly-4/react-core/src/declarations.d.ts

```diff
@@ -6,3 +6,24 @@ declare module 'victory-core' {
   export const Path: any;
   export const TextSize: any;
 }
+
+declare module 'linear-layout-vector' {
```

Comment:
hey @jenny-s51 - my only ounce of feedback so far here would be - can we move this declaration into patternfly-4/react-virtualized-extension some place? I believe `linear-layout-vector` is only a dependency of that package (so probably unnecessary in react-core).

Also, i think you can safely remove package-lock.json as well. We don't ship lock files to consumers (just reference the dependences in the package.json for each package) and all of the dev dependencies for PF React end up getting locked in the root `yarn.lock` file (which should update when you install new packages or run `yarn install` in the root).

Great job w/ this, it's a huge help!!! ⭐️